package.json

```diff
@@ -14,6 +14,11 @@
   "scripts": {
     "test": "jest"
   },
+  "files": [
+    "src",
+    "!src/*.spec.js",
+    "example.js"
```

Comment:
We don't want `example.js` either?

Reply:
Oh I figured maybe we did. I'll remove it.